WTWSMS/history/titles/e_persia.txt

```diff
@@ -8,11 +8,29 @@
 	law = centralization_3
 }
 11.1.1={
-	holder=159546
+	holder=159546 
 }
 51.6.1={
+	holder=159547
```

Comment:
Is this Vologases I of Parthia?

Reply:
> 
> 
> Is this Vologases I of Parthia?

This character was deleted by PDX when they expanded Arsacid genealogy. But because he is named Vologases and he is son of Vonones II (only death date did not match), i identified him with Vologases I.

Reply:
Is it possible to add the expanded Arsacid genealogy from vanilla without many duplicates?

Reply:
> 
> 
> Is it possible to add the expanded Arsacid genealogy from vanilla without many duplicates?

Yes, its possible and i'm going to add vanilla parthian/armenian Arsacids (except Vologases I).

Reply:
Why was this character removed in vanilla?

Reply:
> 
> 
> Why was this character removed in vanilla?

I don't why, but they removed him.

Reply:
I eventually decided to preserve current vanilla Arsacid setup because i didn't wanted to have ID gap.